modules/View.js

```diff
@@ -696,6 +705,55 @@
     //************** Private methods **************//
 
     /**
+     * Attaches views using this.viewMap. The API for viewMap looks like:
+     * viewMap: {
```

Comment:
what about if the injection point is a selector and not inject="" (I think torso supported using something other than inject attribute).

Reply:
It's either the injection site element itself or the [inject='...'] attribute. There's no real good way to let them specify the injection site element in a map like this and I've never seen anyone do it that way anyway. I think this is okay